### ISSUE TITLE
The terminal process "cmd.exe /k" terminated with exit code: 1.

### ISSUE THREAD

**Issue by pixieaka** [microsoft/vscode#118693]
- VS Code Version: 1.54.1
- OS Version: Windows 10 20H2 Build 19042.572

Steps to Reproduce:

1. Open terminal
2. Execute `help` command
3. Execute `exit` command

![1](https://user-images.githubusercontent.com/17777237/110739650-4444b680-823a-11eb-9dbe-dd016326cb77.gif)

### My configuration for terminal
**Settings**
```
  "terminal.integrated.shell.windows": "cmd.exe",
  "terminal.integrated.shellArgs.windows": "/k D:\\Root\\Server\\htdocs\\commands.bat",
```

**commands.bat file content**
```
@echo off


@REM System---------------------------------------------------------------------------

doskey ls=dir /w /ogn /p
doskey ll=dir /a /ogn /p
doskey cp=copy $*
doskey mv=move $*

@REM END System-----------------------------------------------------------------------
```

Does this issue occur when all extensions are disabled?: Yes

**Comment by pixieaka** [comment]
@meganrogge Another thing when I user Command Prompt with this setting
```
"terminal.integrated.shellArgs.windows": "/k D:\\Root\\Server\\htdocs\\commands.bat",
```
Its adds weird padding before it even when I run `cls` command to clear any previous lines

Steps to Reproduce:
1. Change setting to `"terminal.integrated.shell.windows": "cmd.exe",` and open first terminal
2. Remove `"terminal.integrated.shell.windows": "cmd.exe",` to use default Power Shell again
2. Split terminal to get side by side `Command Prompt` and `Power Shell`

![2](https://user-images.githubusercontent.com/17777237/111631572-35db3980-87fc-11eb-9cc0-0c9764f1c585.gif)

**Comment by pixieaka** [comment]
@Tyriar Any info for this issue ?

**Comment by pixieaka** [comment]
@meganrogge Any info about this ?

**Comment by meganrogge** [comment]
![recording (67)](https://user-images.githubusercontent.com/29464607/113182295-978da180-9207-11eb-906f-06ebe8386b09.gif)

I'm unable to reproduce this. When you click `Select Default Shell`, what's the powershell path listed there? Does it reproduce in insider's?

**Comment by meganrogge** [comment]
I can't reproduce the first problem. I can reproduce the line at the top though after clearing

**Comment by pixieaka** [comment]
@meganrogge  My windows version is `Windows 10 20H2 Build 19042.572`. Can you try with this version because first problem is with `shellArgs` or without it.

**Comment by meganrogge** [comment]
I don't have access to that particular version.

**Comment by Milo123459** [comment]
Can't reproduce as a quick FYI. I'm on 20H2 (19042.870)

**Comment by pixieaka** [comment]
@meganrogge @Milo123459  This is latest version 1.55.0 Without any settings unless `cmd.exe` and without any extension.

![2](https://user-images.githubusercontent.com/17777237/113200256-61ccd500-9268-11eb-8ddd-4fe74500c8ec.gif)

**Comment by Milo123459** [comment]
Yup, same. Tested on both. Let me try again

**Comment by Milo123459** [comment]
![img](https://salvage.is-inside.me/ncy0m1yD.png) 
This happens on the latest version.. Not exactly the same

**Comment by meganrogge** [comment]
You can turn off the error notification with `terminal.integrated.confirmOnExit`

**Comment by meganrogge** [comment]
the padding issue also happens on windows terminal, so it's a conpty issue as opposed to a vscode issue.

**Comment by pixieaka** [comment]
@meganrogge Error still shows even with `terminal.integrated.confirmOnExit`.

**Comment by pixieaka** [comment]
![image](https://user-images.githubusercontent.com/17777237/113211174-7368a980-9275-11eb-908f-26da12d01e3b.png)

![image](https://user-images.githubusercontent.com/17777237/113211277-91cea500-9275-11eb-9e71-9935d05cd14e.png)

**Comment by Tyriar** [comment]
Oh that's the wrong setting sorry, it was `terminal.integrated.showExitAlert`
